app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class CartsController < ApplicationController
+
+  def show
+    @carts = Product.joins(:cart)
```

Comment:
Here you should use `session[:cart_id]` and get all `Products` from `Cart` with this `id`.
And it is better name this variable `@products` because it contains products.